onboarding/time-management.md

```diff
@@ -0,0 +1,201 @@
+- **Plan Your Meals**: Schedule your meals for the day. Plan ahead, so you can buy any missing ingredients, make better food choices, and save mental energy for important tasks like learning new coding topics.
+- **Schedule Fun**: Include enjoyable activities throughout the week. Even 30 minutes dedicated to a hobby can help recharge you and bring joy.
+
+---
+
+### Tackling Procrastination
+
+Why do people procrastinate? Here are some common reasons:
+
+- Perfectionism/fear of negative evaluation
+- The need for time pressure
+- The issue seems overwhelming
+- Difficulty finding a starting point
+- A task is not appealing
+- Wanting a big chunk of time
+
+One key strategy for overcoming procrastination is to identify and challenge the unhelpful automatic thoughts that make tasks feel more overwhelming or unrealistic.
+
+**Challenging Unhelpful Automatic Thinking**
+
+Overcoming procrastination often involves challenging overly optimistic and unhelpful automatic thinking. These cognitive distortions can make tasks feel more daunting or unrealistic than they really are.
+
+
+### Challenging Cognitive Distortions:
+
+1. **Is this thought really true? What evidence or exceptions are there?**
+2. **Is this thought helpful?**
+3. **How else can I think about this?**
+4. **What would I say to a friend in this situation?**
+5. **Am I making assumptions or generalizations?**
+
+### Example Situation:
+
+**Situation**: Working on a difficult ticket  
+
+**Automatic thoughts**:  
+- *"I have to do all of this today. I must do it perfectly. If I don’t finish, my boss will be mad.
+If my work isn’t perfect and timely, my boss will be mad and I’ll be fired.
+"*
+
+**Feelings and intensity (1-10)**:  
+- 8 (Overwhelmed, stressed)
+
+**Cognitive distortions**:
+- **All-or-nothing thinking**: "I have to finish everything at once. "
+- **Catastrophizing**: "If I don’t do this perfectly, it's going to be a disaster."
+
+**Rational response**:  
+- *"It’s not the end of the world if I can’t do it all today. I can at least get started today and finish tomorrow if needed. I can ask my boss for help. My boss has been understanding in the past and could be again. I have received good feedback on other work, so it’s unlikely I’ll lose my job if this one isn’t perfect."*
+
+
+### Using the Woebot App for Cognitive Distortions:
+
+You can go through this whole process and learn about cognitive distortions using the Woebot app.
+
+##  Common Mistakes / Misconceptions
+### Multitasking is a Myth!
+
+Did you know that multitasking is actually a myth?
+
+The costs of multitasking include:
+
+- **Extra time:** Switching between tasks takes longer than you think.
+- **Quality:** Trying to do too much at once lowers the quality of your work.
+- **Stress:** Constant task-switching can create added stress.
+
+##  Supplemental Materials
+There are a lot of amazing time management tips that come from 
+behavioral ADHD advice — check out these resources!
```

Comment:
```suggestion
There are a lot of amazing time management tips that come from behavioral ADHD advice — check out these resources!
```

Reply:
Hi @mai-repo can you leave a comment or even a commit hash clarifying where this has been addressed? Please do this before resolving each change request